lake/api/querydriver.go

```diff
@@ -1,41 +1,29 @@
 package api
```

Comment:
`querydriver.go` no longer makes sense as a name for this file. Move the contents to `lake/api/api.go` since that's the only file in which they're used and both files are small?